druuid.rb

```diff
@@ -38,6 +38,20 @@ def time uuid, epoch_offset = epoch
       Time.at (ms / 1e3) + epoch_offset.to_i
     end
 
+    # Provides lower and upper bound ids based on datetime range provided.
+    #
+    # @param [Time] time of lower bound
+    # @param [Time] time of upper bound
+    # @return [Array] ids of lower and upper bound range
+    # @example
+    #   Druuid.id_bounds Time.utc(2015,2,3,12,2,34), Time.utc(2015,2,3,17,56,12)
+    #   # => [11936695196844032000, 11936873186336964607]
+    def id_bounds lower_datetime, upper_datetime
```

Comment:
What do you think about making the second param optional and just having it default to the lower if it's not provided?